cmd/metal-api/internal/service/v1/audit.go

```diff
@@ -65,6 +67,13 @@ func NewAuditResponse(e auditing.Entry) *AuditResponse {
 		body = v
 	case []byte:
 		body = string(v)
+	default:
+		b, err := json.Marshal(v)
+		if err == nil {
+			body = string(b)
+		} else {
+			body = fmt.Sprintf("unknown body: %v", err)
```

Comment:
maybe the payload should also be put into the error ?

Reply:
Good point. I'll replace the err with the value itself as it does not add relevant information over the data itself.